src/messages/ui.en.json

```diff
@@ -156,12 +154,16 @@
   "connectMB.connectCable.heading": "Connect USB cable to micro:bit",
   "connectMB.connectCable.subtitle": "Connect the micro:bit to this computer with a USB cable so that the machine learning tool program can be downloaded to it.",
 
+  "connectMB.connectCable.altText": "animation showing a USB cable being connected to the top of a micro:bit",
   "connectMB.connectCable.skip": "Skip: program already downloaded?",
 
   "connectMB.webPopup": "Select micro:bit",
   "connectMB.webPopup.instruction.heading": "In the next popup",
   "connectMB.webPopup.instruction1": "Choose your micro:bit",
-  "connectMB.webPopup.instruction2": "Select 'Connect'",
+  "connectMB.webPopup.webUsb.instruction2": "Select 'Connect'",
+  "connectMB.webPopup.webBluetooth.instruction2": "Select 'Pair'",
+  "connectMB.webPopup.webUsb.altText": "WebUSB connection dialog with BBC micro:bit selected",
```

Comment:
Python Editor uses:

WebUSB connection dialog with BBC micro:bit entry labelled 1 and Connect button labelled 2

Maybe we should match and do similarly for Bluetooth?

Reply:
I would have copied that exactly except that we haven't put the arrows / labels on/over the image yet.

Reply:
Ah yeah. Still though, I think we may as well just get the alt text right and then do the arrows when we get to them.

Reply:
I went for the optimistic approach that we'll do the labels [9123dc9](https://github.com/microbit-foundation/ml-trainer/pull/125/commits/9123dc94cdbac5acdd1ef801990698cfb972cbec).